install.sh

```diff
@@ -269,9 +269,20 @@ check_path() {
 	if gum confirm "create /usr/local/bin/tea?" --affirmative="make symlink" --negative="skip"
 	then
 		echo  #spacer
-		sudo mkdir -p /usr/local/bin
-		sudo ln -sf "$tea" /usr/local/bin/tea
 
+		if [ "$USER" = "root" ] || which sudo >/dev/null;
+		then
+			sudo mkdir -p /usr/local/bin
```

Comment:
well if the user is `root` presumably we should just not use `sudo` since the `if` checks for either and `sudo` may not be there in fact.

Reply:
Fixed in recent commit!